print-routes.js

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env node
+
+'use strict'
+
+require('dotenv').config()
+
+const assert = require('assert')
+const parseArgs = require('./args')
+const log = require('./log')
+const { exit, requireFastifyForModule, requireServerPluginFromPath, showHelpForCommand } = require('./util')
+
+let Fastify = null
+
+function loadModules (opts) {
+  try {
+    Fastify = requireFastifyForModule(opts._[0]).module
+  } catch (e) {
+    module.exports.stop(e)
+  }
+}
```

Comment:
I think this can be abstracted away and shared with start.

Reply:
Like this?

```js
// util.js
function tryRequireFastifyForModuleOrExit(modulePath) {
  try {
    return requireFastifyForModule(modulePath);
  } catch (e) {
    exit(e);
  }
}
```

Reply:
I was wrong. It's fine as it is.